packages/headless/src/controllers/search-box/headless-search-box.ts

```diff
@@ -140,6 +141,32 @@ export function buildSearchBox(
       performSearch(logSearchboxSubmit());
     },
 
+    /**
+     * Add delimiters to a highlighted suggestion.
+     *
+     * @param highlightedSuggestion The highlighted suggestion.
+     * @param openingMatchDelimiter The opening delimiter for a match in the suggestion string (e.g. '<strong>')
+     * @param closingMatchDelimiter The closing delimiter for a match in the suggestion string (e.g. '</strong>')
+     * @param openingCorrectedDelimiter The opening delimiter for a correction in the suggestion string (e.g. '<i>')
+     * @param closingCorrectedDelimiter The closing delimiter for a correction in the suggestion string (e.g. '</i>')
+     */
+
+    formatHighlightedSuggestion(
```

Comment:
Could we provide good defaults for this method? e.g. the examples that are provided

Reply:
Previous comment could apply to the method in `highlight.ts`

Here, you could only pass the reference to that method e.g.:
```typescript
... 

/**
 * Add delimiters to a highlighted suggestion.
 *
 * @param highlightedSuggestion The highlighted suggestion.
 * @param openingMatchDelimiter The opening delimiter for a match in the suggestion string (e.g. '<strong>')
 * @param closingMatchDelimiter The closing delimiter for a match in the suggestion string (e.g. '</strong>')
 * @param openingCorrectedDelimiter The opening delimiter for a correction in the suggestion string (e.g. '<i>')
 * @param closingCorrectedDelimiter The closing delimiter for a correction in the suggestion string (e.g. '</i>')
 */

formatHighlightedSuggestion,

...
```

Reply:
I would rename this method to `getHighlightedSuggestions` and have it retrieve the suggestions from state versus let it accept a suggestion.

```
getHighlightedSuggestions(options): string[] {}
```

Another possibility which might be even better would be to move the highlighting options to searchBox options object passed when initializing the controller.

This way, the value in state will already be highlighted, removing the need to call this function on every subscribe callback call. This also makes it possible to discover all options in one place, and simplifies the search box api by removing this method. I think this is the way to go,

Reply:
> Another possibility which might be even better would be to move the highlighting options to searchBox options object passed when initializing the controller.

👍  This seems better indeed !